src/include/sof/audio/dai_copier.h

```diff
@@ -52,4 +52,7 @@ int dai_zephyr_get_hw_params(struct dai_data *dd, struct comp_dev *dev,
 int dai_zephyr_multi_endpoint_copy(struct dai_data **dd, struct comp_dev *dev,
 				   struct comp_buffer *multi_endpoint_buffer,
 				   int num_endpoints);
+
+int dai_zephyr_unbind(struct dai_data *dd, struct comp_dev *dev, void *data);
+
```

Comment:
extra blank line add

Reply:
@btian1 Oops, forgot to check this. But hmm, there's traditionally blank link between last line of header declaration and "#endif" (this is used in Zephyr and Linux so I'd say let's follow same policy). We seem to have a few exceptions in SOF, but these should be fixed.